extensions/PhabBugz/lib/Feed.pm

```diff
@@ -384,6 +391,41 @@ sub process_revision_change {
   );
   INFO($log_message);
 
+  # Change is a submission of the uplift request form.
+  if (_is_uplift_request_form_change($story_text)) {
+    my $revision_phid = $revision->phid;
+    INFO(
+      "Uplift request form submitted on $revision_phid, " .
+      "requesting `#release-managers` review."
+    );
+
+    # Get `#release-managers` review group.
+    my $release_managers_phid = Bugzilla::Extension::PhabBugz::Project
+      ->new_from_query({name => 'release-managers'})
+      ->phid;
+
+    # Request `#release-managers` review on each revision.
+    foreach my $stack_revision_phid (keys %{$revision->{stack_graph_raw}}) {
```

Comment:
Since you added this to Revision.pm as an attribute then you can just access it as a normal method.
It is preferred over accessing the internal data directly.


    foreach my $stack_revision_phid (keys %{$revision->stack_graph_raw}) {